style.css

```diff
@@ -0,0 +1,572 @@
+@import url("https://fonts.googleapis.com/css?family=Lato");
+
+* {
+  font-family: "Lato", sans-serif;
+}
+
+.paragraph {
+  font-family: "Lato", sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 17px;
+  letter-spacing: -0.01em;
+  text-align: left;
+  width: 328px;
+  height: 51px;
+  color: #000;
+}
+
+.theme {
+  background-color: #ec5242;
+  background-image: url('./images/yellow.jpg');
+  -webkit-text-fill-color: rgba(0, 0, 0, 0);
+  color: transparent;
+  -webkit-background-clip: text;
+  font-family: "Lato", sans-serif;
+}
+
+.date {
+  font-family: "Lato", sans-serif;
+  font-size: 20px;
+  font-weight: 900;
+  letter-spacing: -0.08em;
+  text-align: left;
+  color: #fff;
+}
+
+.venue {
+  font-family: "Lato", sans-serif;
+  font-size: 13px;
+  font-weight: 700;
+  letter-spacing: -0.02em;
+  text-align: left;
+  color: #fff;
+}
+
+.main {
+  width: 95%;
+  height: 1080px;
+  top: 549px;
+  padding: 11px 12px 11px 12px;
+  gap: 10px;
+  background-color: #272727;
+  background-image: url("./images/abstractbg.svg");
+  margin-top: -145px;
+}
+
```

Comment:
- [ ] `required`
- [ ] Please make sure the responsive is working fine in the mobile version
- [ ] please screenshots below

`your template`
![image](https://github.com/Danny-imo/Capstone-project/assets/66230306/84b47fb4-e6a3-423c-a8ea-851c163941f9)

`original tempalate`
![image](https://github.com/Danny-imo/Capstone-project/assets/66230306/8509022f-36d3-4a49-903d-2f975a564b15)